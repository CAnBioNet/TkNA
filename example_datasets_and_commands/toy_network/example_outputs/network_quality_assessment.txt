PUC: 0.0%
Mean degree: 2.2857142857142856
Edges in network over the edges in a full graph: 0.38
Positive:negative edge ratio: 1.0

Expected positive:negative edge ratio in full graph: 0.75
Deviation from expected positive:negative edge ratio in full graph: 0.33

Number of positive nodes: 4
Number of negative nodes: 3
Number of total nodes: 7

Number of positive edges: 4
Number of negative edges: 4
Number of total edges: 8
