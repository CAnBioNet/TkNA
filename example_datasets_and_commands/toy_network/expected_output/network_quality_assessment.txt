PUC: 36.11%
Mean degree: 4.857142857142857
Edges in network over the edges in a full graph: 0.81
Positive:negative edge ratio: 0.8888888888888888

Expected positive:negative edge ratio in full graph: 1.1
Deviation from expected positive:negative edge ratio in full graph: -0.19

Number of positive nodes: 2
Number of negative nodes: 5
Number of total nodes: 7

Number of positive edges: 8
Number of negative edges: 9
Number of total edges: 17
